save-cloud-charts/save-cloud/templates/backend-deployment.yaml

```diff
@@ -22,13 +22,21 @@ spec:
             {{- include "spring-boot.common.env" (merge (dict "service" .Values.backend) .) | nindent 12 }}
             - name: DATABASE_SECRETS_PATH
               value: {{ .Values.backend.dbPasswordFile }}
+            - name: JAVA_TOOL_OPTIONS
+              value: -XX:ReservedCodeCacheSize=48M
           volumeMounts:
             - {{ include "spring-boot.config-volume-mount" . | indent 14 | trim }}
             - name: fs-storage
               mountPath: /home/cnb/files
             - name: database-secret
               mountPath: {{ .Values.backend.dbPasswordFile }}
           {{- include "spring-boot.management" .Values.backend | nindent 10 }}
+          resources:
+            limits:
+              memory: 500M
```

Comment:
How did you set these limits? Randomly? :)

Reply:
No, we have data about memory consumption on our current server, it doesn't go above 500M.